README.md

```diff
@@ -29,9 +29,14 @@ pip install -r requirements.txt
 ### 5. Run the Script
 In the same command prompt or Terminal window, type the following, and press enter:
 ```
-python geo_heatmap.py <file>
+python geo_heatmap.py <file> yyyy month day yyyy month day
```

Comment:
I think it'd make more sense to make two optional named arguments like:

```
--min-date=yyyy-mm-dd
--max-date=yyyy-mm-dd
```

iso date format with hyphens